### PR TITLE
Node modules

### PR DESCRIPTION
**Pull request by noliver-ykc**
For some reason the node modules backend wasnt in gitignore so i added that (it kept stagging everything for me)

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -2,6 +2,9 @@
 10_data.sql
 *.log
 
+/backend/node_modules/
```
Comment (mutokrm) [inline backend/.gitignore]: Since the backend is written in Go, I think the backend folder shouldn't have `node_modules` 🤔

Comment (noliver-ykc) [reply backend/.gitignore]: OMG THAT MAKES SO MUCH SENSE

Comment (noliver-ykc) [reply backend/.gitignore]: i think i added it by mistake a long time ago but never realized
thanks for pointing that out

Comment (noliver-ykc) [reply backend/.gitignore]: ill just delete it haha

Comment (mutokrm) [reply backend/.gitignore]: Yay! Problem solved!

**Pull request closed without merging**